### PR TITLE
77 age sex index

### LINKED ISSUE #77: age-sex index
Hi @edarin,

I’m trying to translate an age-sex model into the pipeline you created. However, when I attempt to create the index for age-sex using https://github.com/OxfordDemSci/UkraineNowPop/blob/f666404ca7cbaace5e669f251b9c0b9362482ff0/src/1_pop_data/10_create_master_index.R, the produced indexes do not seem to match the ages (see first image below), and each age index is produced twice for the same sex (see second image below). I might be doing something wrong. Could you check if you get the same results or if anything needs to be adjusted?

image

image

Thanks. 

Andrea Aparicio

### PR DESCRIPTION
**Pull request by edarin**

### CODE REVIEW

File: src/1_pop_data/10_create_master_index.R [hunk 1]
```diff
@@ -65,9 +67,15 @@ time_index <- time_index_expanded |>
...
+agesex_index <- lapply(
+  agesex, function(agesex_) {
+    agesex_col_to_query_args(agesex_) |>
+      as_tibble() |>
+      mutate(agesex = agesex_)
+  }
+) |>
   bind_rows() |>
-  arrange(age_min) |>
+  arrange(age_min, desc(agesex)) |>
```
Comment (alaparicioc) [inline src/1_pop_data/10_create_master_index.R:78]: I cannot comment on the line of code, but s should be integer, not numeric. Something like s = s_name |> as_factor() |> as.integer(),

**Review by alaparicioc — CHANGES_REQUESTED** [review]
__I cannot comment on the line of code, but s should be integer, not numeric. Something like s = s_name |> as_factor() |> as.integer(),

File: src/1_pop_data/10_create_master_index.R (hunk 1 above)
Comment (edarin) [reply src/1_pop_data/10_create_master_index.R:78]: Done! https://github.com/OxfordDemSci/UkraineNowPop/commit/448227b01b1a2b587fb18337719b6b7c2d060806
Weird that you cant commit change?
Sorry for the cumbersome process. Do you want to check together why you cant commit?

Comment (alaparicioc) [reply src/1_pop_data/10_create_master_index.R:78]: thansl! At some point,we can check togther how I am commiting. It may be online though.

**Pull request merged**